### ISSUE TITLE
Страница со ссылками на другие городские дашборды

### ISSUE THREAD

**Issue by davojta** [opendataby/city-dashboard#7]
Нужно сделать страницу со ссылками и скриншотами других дашбордов, чтобы было понятно о чем проект

Ссылки можно найти ниже

https://www.portlandoregon.gov/dashboard/66479

**Comment by davojta** [comment]
http://citydashboard.be.unsw.edu.au/
https://dashboard.edmonton.ca/ - one of the best

**Comment by davojta** [comment]
https://performance.lacity.org/ - one more example

**Comment by davojta** [comment]
https://www.portlandoregon.gov/dashboard/

**Comment by abitrolly** [comment]
Да, мне кажется надо не с Минска начинать. Здесь всё гниловато в плане открытости госов. =)

**Comment by davojta** [comment]
https://dashboard.edmonton.ca/green - тут интересная легенда, 
типо нужно улучшать, всё хорошо, близкая цель, нужны данные \ больше измерений, происходят измерения

**Comment by KonstantinReido** [comment]
@davojta давай такую информацию в вики агрегировать, там уже есть список ссылок

@abitrolly что ты предлогаешь? Какой город?

**Comment by abitrolly** [comment]
@KonstantinReido любой в РБ, где кто-то есть неравнодушный.

**Comment by davojta** [comment]
Мне кажется, нам надо сделать отдельную страничку в проекте и туда скинуть ссылки со скриншотами, чтобы у всех было понимание что это за проект
